how-to/workspace-platform-starter/client/src/framework/bootstrapper.ts

```diff
@@ -27,6 +28,19 @@ const logger = createLogger("Bootstrapper");
 let bootstrapOptions: BootstrapOptions | undefined;
 let deregistered = false;
 
+/**
+ * Used to register home related actions.
+ * @returns Nothing
+ */
+async function registerHomeSupportedActions() {
```

Comment:
This method need not be async, also there is supposed to be a lint rule that enforce return types, but that doesn't appear to be working

Reply:
updated